runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -121,7 +121,7 @@ native2InterpJavaUpcallF(J9UpcallMetaData *data, void *argsListPointer)
 double JNICALL
 native2InterpJavaUpcallD(J9UpcallMetaData *data, void *argsListPointer)
 {
-	UDATA returnValue = native2InterpJavaUpcallImpl(data, argsListPointer);
+	U_64 returnValue = native2InterpJavaUpcallImpl(data, argsListPointer);
 	/* The value returned from the upcall method is literally the double precision (64bit) IEEE 754 floating-point
```

Comment:
Please a the missing hyphen in "64-bit".

Reply:
Fixed.